scripts/tablet-mon.py

```diff
@@ -447,7 +448,8 @@ def tablet_id_for_table(table_id):
         return ret
 
     for tablet in session.execute(tablets_query):
-        id = (tablet.table_id, tablet.last_token, tablet_id_for_table(tablet.table_id))
+        tablet_seq = tablet_id_for_table(tablet.table_id)
+        id = (tablet.table_id, tablet.last_token)
         replicas = set(tablet.replicas)
```

Comment:
Ah, looks like we needed tablet comparison stability to detect preexisting tablets. Thanks.